common/presenters/allocations-to-table-component.js

```diff
@@ -27,11 +27,15 @@ function _byAdded(totalSlots, unfilledSlots) {
 function allocationsToTableComponent({
   showFromLocation = false,
   showRemaining = false,
+  query,
```

Comment:
There seems to be no test to ensure that this value is passed correctly to the `objectToTableHead` presenter. It would be good to add something around that so if this property is removed the tests would fail.